src/content/developers/docs/consensus-mechanisms/pow/mining-algorithms/dagger-hashamoto/index.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Dagger-Hashamoto
+description: A detailed look at the Dagger-Hashamoto algorithm.
+lang: en
+sidebar: true
+incomplete: false
+---
+
+This page includes detailed information about the Dagger Hashamoto mining algorithm. Through a series of substanbtial upgrades, this algorithm was superceded by [Ethash](/ethash). However, this original version has historical significance and is included here because it was an important innovation in Ethereum's development. Proof-of-work mining itself will be deprecated in favour of proof-of-stake during the merge which is expected to happen in summer 2022. To understand the information in this page it is recommended to first read our introductory pages on [proof-of-work consensus](/developers/docs/consensus-mechanisms/pow) and [mining](/developers/docs/consensus-mechanisms/pow/mining).
+
+Dagger Hashimoto was a precursor research implementation and specification for the mining algorithm for Ethereum, while it has been superceded by [Ethash](#ethash). The information on this page is for historical interest - it is not up to date with current Ethereum mining.
+
+Dagger Hashimoto aims to simultaneously satisfy two goals:
+
+1.  **ASIC-resistance**: the benefit from creating specialized hardware for the algorithm should be as small as possible, ideally to the point that even in an economy where ASICs have been developed the speedup is sufficiently small that it is still marginally profitable for users on ordinary computers to mine with spare CPU power.
```

Comment:
This gets kind of hard to read, do you think cutting this at the first sentence retains the sentiment here?

```suggestion
1.  **ASIC-resistance**: the benefit from creating specialized hardware for the algorithm should be as small as possible
```